practice-two/src/components/FormInput/FormInput.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import './FormInput.css';
+import PropTypes from 'prop-types';
+import { MESSAGES, REGEX } from '@constants';
+
+const FormInput = (props) => {
+  const [focused, setFocused] = useState(false);
+  const { label, errorMessage, onChange, ...inputProps } = props;
+
+  const handleFocus = () => {
+    setFocused(true);
+  };
+
+  return (
+    <div className='form__input'>
+      <label className='input__label'>{label}</label>
+      <input
+        className='input__field'
+        {...inputProps}
+        onChange={onChange}
+        onBlur={handleFocus}
+        // eslint-disable-next-line react/no-unknown-property
+        focused={focused.toString()}
+      />
+      {errorMessage && errorMessage !== '' && (
```

Comment:
`errorMessage` is enough

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/d0e304e6-4786-4092-ad65-d6da9f8f4fd4)